lib/chef/resource/aws_route_table.rb

```diff
@@ -35,6 +35,16 @@ class Chef::Resource::AwsRouteTable < Chef::Provisioning::AWSDriver::AWSResource
   attribute :vpc,    kind_of: [ String, AwsVpc, AWS::EC2::VPC ], required: true
 
   #
+  # Enable route propagation from one or more virtual private gateways
+  #
+  # The value should be an array of virtual private gateway ID, like so:
+  # ```ruby
+  # vgw_route_propagation ['vgw-abcd1234', 'vgw-abcd5678']
+  # ```
+  #
+  attribute :vgw_route_propagation, kind_of: Array
```

Comment:
I could see us creating resources for `aws_virtual_private_gateway` and `aws_customer_gateway` in the future.  Because of this I would like this attribute renamed to `:virtual_private_gateway`.  Then the attribute would be named the same for route tables and VPCs.

For niceness, you should also have `kind_of: [String, Array]` - If they provide a string you can just turn it into an array with `virtual_private_gateway = [new_resource.virtual_private_gateway].flatten` in the provider.